### DISCUSSION TITLE
Using a NPM package that has / in name?

### DISCUSSION THREAD

**Discussion by MirilRetry** [denoland/deno#26686] [Q&A] [answered]
I'm trying to use the package [@paddlejs-models/ocr](https://www.npmjs.com/package/@paddlejs-models/ocr), but when I attempt to import it using:
`import * as ocr from 'npm:paddlejs-models/ocr';`
I get an error stating that the paddlejs-models package does not exist. This is correct, as the project name contains "/ocr." How can I properly import it?

**Comment by marvinhagemeister** [comment, marked as answer]
The error message is correct. The package `paddlejs-models` doesn't exist on npm. I think you meant to use `@paddlejs-models/ocr` instead which is a valid package name. The format on npm for these is `@<scope>/<name>`. You cannot leave out the `@` character though, same as in Node. Scoped package names _must_ always start with the `@` character.

```diff
- import * as ocr from 'npm:paddlejs-models/ocr';
+ import * as ocr from 'npm:@paddlejs-models/ocr';
```

**Comment by MirilRetry** [comment]
Thank you! Adding @ fixed it.
